src/seedu/addressbook/data/person/Phone.java

```diff
@@ -35,6 +38,9 @@ public Phone(String phone, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidPhone(String test) {
         return test.matches(PHONE_VALIDATION_REGEX);
     }
+    public List<String> getPhone() {
+        return Arrays.asList(value.split("\\s+"));
```

Comment:
According to the regex, the phone number can only be a string of digits. Then, there is no need to split on whitespace (since there will be none). Also, is it necessary to return the results in a list? Since there will only be 1 string.